{
  "cve": "CVE-2022-27448",
  "mitre": {
    "cpes": [],
    "created": "2022-04-14T12:56:55+00:00",
    "description": "There is an Assertion failure in MariaDB Server v10.9 and below via 'node->pcur->rel_pos == BTR_PCUR_ON' at /row/row0mysql.cc.",
    "metrics": {
      "cvssV2_0": {},
      "cvssV3_0": {},
      "cvssV3_1": {},
      "cvssV4_0": {}
    },
    "mitre_repo_path": "cves/2022/27xxx/CVE-2022-27448.json",
    "references": [
      "https://jira.mariadb.org/browse/MDEV-28095",
      "https://lists.debian.org/debian-lts-announce/2022/09/msg00023.html",
      "https://security.netapp.com/advisory/ntap-20220526-0006/"
    ],
    "title": null,
    "updated": "2024-08-03T05:25:32.683000+00:00",
    "vendors": [],
    "weaknesses": []
  },
  "nvd": {
    "cpes": [
      "cpe:2.3:a:mariadb:mariadb:*:*:*:*:*:*:*:*",
      "cpe:2.3:o:debian:debian_linux:10.0:*:*:*:*:*:*:*"
    ],
    "created": "2022-04-14T13:15:11.950000+00:00",
    "description": "There is an Assertion failure in MariaDB Server v10.9 and below via 'node->pcur->rel_pos == BTR_PCUR_ON' at /row/row0mysql.cc.",
    "metrics": {
      "cvssV2_0": {
        "score": 5.0,
        "vector": "AV:N/AC:L/Au:N/C:N/I:N/A:P"
      },
      "cvssV3_0": {},
      "cvssV3_1": {
        "score": 7.5,
        "vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
      },
      "cvssV4_0": {}
    },
    "nvd_repo_path": "2022/CVE-2022-27448.json",
    "references": [
      "https://jira.mariadb.org/browse/MDEV-28095",
      "https://lists.debian.org/debian-lts-announce/2022/09/msg00023.html",
      "https://security.netapp.com/advisory/ntap-20220526-0006/"
    ],
    "title": null,
    "updated": "2022-10-08T03:15:08.320000+00:00",
    "vendors": [
      "debian",
      "debian$PRODUCT$debian_linux",
      "mariadb",
      "mariadb$PRODUCT$mariadb"
    ],
    "weaknesses": [
      "CWE-617"
    ]
  },
  "opencve": {
    "changes": [],
    "cpes": {
      "data": [
        "cpe:/a:redhat:enterprise_linux:8",
        "cpe:/a:redhat:enterprise_linux:9",
        "cpe:/a:redhat:rhel_software_collections:3::el7",
        "cpe:2.3:a:mariadb:mariadb:*:*:*:*:*:*:*:*",
        "cpe:2.3:o:debian:debian_linux:10.0:*:*:*:*:*:*:*"
      ],
      "providers": [
        "nvd",
        "redhat"
      ]
    },
    "created": {
      "data": "2022-03-16T00:00:00+00:00",
      "provider": "redhat"
    },
    "description": {
      "data": "There is an Assertion failure in MariaDB Server v10.9 and below via 'node->pcur->rel_pos == BTR_PCUR_ON' at /row/row0mysql.cc.",
      "provider": "mitre"
    },
    "metrics": {
      "cvssV2_0": {
        "data": {
          "score": 5.0,
          "vector": "AV:N/AC:L/Au:N/C:N/I:N/A:P"
        },
        "provider": "nvd"
      },
      "cvssV3_0": {
        "data": {},
        "provider": null
      },
      "cvssV3_1": {
        "data": {
          "score": 7.5,
          "vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
        },
        "provider": "nvd"
      },
      "cvssV4_0": {
        "data": {},
        "provider": null
      },
      "kev": {
        "data": {},
        "provider": null
      },
      "ssvc": {
        "data": {},
        "provider": null
      },
      "threat_severity": {
        "data": "Moderate",
        "provider": "redhat"
      }
    },
    "references": {
      "data": [
        "https://jira.mariadb.org/browse/MDEV-28095",
        "https://lists.debian.org/debian-lts-announce/2022/09/msg00023.html",
        "https://nvd.nist.gov/vuln/detail/CVE-2022-27448",
        "https://security.netapp.com/advisory/ntap-20220526-0006/",
        "https://www.cve.org/CVERecord?id=CVE-2022-27448"
      ],
      "providers": [
        "mitre",
        "nvd",
        "redhat"
      ]
    },
    "title": {
      "data": "mariadb: crash in multi-update and implicit grouping",
      "provider": "nvd"
    },
    "updated": {
      "data": "2024-08-03T05:25:32.683000+00:00",
      "provider": "mitre"
    },
    "vendors": {
      "data": [
        "debian",
        "debian$PRODUCT$debian_linux",
        "mariadb",
        "mariadb$PRODUCT$mariadb",
        "redhat",
        "redhat$PRODUCT$enterprise_linux",
        "redhat$PRODUCT$rhel_software_collections"
      ],
      "providers": [
        "nvd",
        "redhat"
      ]
    },
    "weaknesses": {
      "data": [
        "CWE-617"
      ],
      "providers": [
        "nvd",
        "redhat"
      ]
    }
  },
  "redhat": {
    "cpes": [
      "cpe:/a:redhat:enterprise_linux:8",
      "cpe:/a:redhat:enterprise_linux:9",
      "cpe:/a:redhat:rhel_software_collections:3::el7"
    ],
    "created": "2022-03-16T00:00:00+00:00",
    "description": "There is an Assertion failure in MariaDB Server v10.9 and below via 'node->pcur->rel_pos == BTR_PCUR_ON' at /row/row0mysql.cc.",
    "metrics": {
      "cvssV2_0": {},
      "cvssV3_0": {},
      "cvssV3_1": {
        "score": 7.5,
        "vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
      },
      "threat_severity": "Moderate"
    },
    "redhat_repo_path": "2022/CVE-2022-27448.json",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2022-27448",
      "https://www.cve.org/CVERecord?id=CVE-2022-27448"
    ],
    "title": "mariadb: crash in multi-update and implicit grouping",
    "updated": "2022-03-16T00:00:00+00:00",
    "vendors": [
      "redhat",
      "redhat$PRODUCT$enterprise_linux",
      "redhat$PRODUCT$rhel_software_collections"
    ],
    "weaknesses": [
      "CWE-617"
    ]
  }
}